src/main/java/com/bergerkiller/bukkit/tc/signactions/SignActionTicket.java

```diff
@@ -42,13 +44,8 @@ public void execute(SignActionEvent info) {
         }
 
         if ((info.hasMember() && info.isPowered())) {
-            double money;
-            if (info.getLine(3).isEmpty()) {
-                money = 30.0;
-            } else {
-                money = ParseUtil.parseDouble(info.getLine(3), 0.0);
-            }
             final String mode = info.getLine(2);
+            final double money = info.getLine(3).isEmpty() ? 0 : ParseUtil.parseDouble(info.getLine(3), 0.0);
```

Comment:
Pretty sure `ParseUtil.parseDouble(info.getLine(3), 0.0)` will still return 0 when the string is empty, so this might be redundant.

Reply:
This is more a fallback for when the user enters text instead of a number

Reply:
`ParseUtil.parseDouble("", 0.0);` will still return 0, that check for an empty string is unnecessary.

Reply:
Fixed in #cdafc647